lib/components/artist/articles/index.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+'use strict';
+
+import Relay from 'react-relay';
+import React from 'react-native';
+const { ScrollView, StyleSheet, View } = React;
+
+import colors from '../../../../data/colors';
+import SerifText from '../../text/serif';
+import ImageView from '../../opaque_image_view';
+import Article from './article';
+
+class Articles extends React.Component {
+  render() {
+    const articles = this.props.articles
+    return (
+      <View style={styles.container}>
+        <SerifText style={styles.heading}>Featured Articles</SerifText>
+        <ScrollView horizontal={true} style={{ overflow: 'visible' }}>
+          { articles.map(article => <Article key={article.slug} article={article} />) }
```

Comment:
Ok, I see, good link that one. Ok, but maybe use id instead of slug, because slugs can be long as they are meant to be human-readable.